only a single inner field is allowed when producing a class

The [basic usage](https://github.com/microsoft/com-rs/blob/master/docs/basic-usage.md#classes) docs say that you can have many inner fields, but adding the `hi` field below does not compile. (Easy enough to work around by adding another struct.) Am I doing it right?

```
use com::interfaces::IUnknown;
use com::sys::{HRESULT,S_OK};

com::interfaces! {
    #[uuid("EFF8970E-C50F-45E0-9284-291CE5A6F771")]
    pub unsafe interface IAnimal: IUnknown {
        fn Eat(&self) -> HRESULT;
    }
}
com::class! {
    pub class BritishShortHairCat: IAnimal {
        hello: u32,
        hi: u32
    }

    impl IAnimal for  BritishShortHairCat {
        fn Eat(&self) -> HRESULT {
            println!("Wotcher");
            S_OK
        }
    }
}
```

results in 

```
error: comparison operators cannot be chained
  --> src\main.rs:13:1
   |
13 | / com::class! {
14 | |     pub class BritishShortHairCat: IAnimal {
15 | |         hello: u32,
16 | |         hi: u32
...  |
24 | |     }
25 | | }
   | |_^
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
help: use `::<...>` instead of `<...>` to specify type arguments
   |
13 | ::com::class! {
   | ^^

error[E0061]: this function takes 2 arguments but 1 argument was supplied
  --> src\main.rs:13:1
   |
13 |   com::class! {
   |   ^----------
   |   |
   |  _defined here
   | |
14 | |     pub class BritishShortHairCat: IAnimal {
15 | |         hello: u32,
16 | |         hi: u32
...  |
24 | |     }
25 | | }
   | | ^
   | | |
   | |_supplied 1 argument
   |   expected 2 arguments
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0061`.
error: could not compile `comtest`
```

--- a/macros/support/src/class/class_factory.rs
+++ b/macros/support/src/class/class_factory.rs
@@ -30,7 +30,7 @@ pub fn generate(class: &Class) -> TokenStream {
                         return ::com::sys::CLASS_E_NOAGGREGATION;
                     }
 
-                    let instance = #class_name::allocate(#(#user_fields)*,);
+                    let instance = #class_name::allocate(#(#user_fields),*);
                     instance.QueryInterface(riid, ppv)
                 }
 

--- a/tests/ui/pass/class_fields.rs
+++ b/tests/ui/pass/class_fields.rs
@@ -1,0 +1,28 @@
+com::interfaces! {
+    #[uuid("12345678-1234-1234-1234-12345678ABCD")]
+    pub unsafe interface ISomething: com::interfaces::iunknown::IUnknown {}
+}
+
+com::class! {
+    pub class ClassOfZero: ISomething {
+    }
+
+    impl ISomething for SomeClass {}
+}
+com::class! {
+    pub class ClassOfOne: ISomething {
+        one: u32,
+    }
+
+    impl ISomething for SomeClass {}
+}
+com::class! {
+    pub class ClassOfTwo: ISomething {
+        one: u32,
+        two: u32
+    }
+
+    impl ISomething for SomeClass {}
+}
+
+fn main() {}

